src/core_codemods/requests_verify.py

```diff
@@ -23,12 +23,18 @@ class RequestsVerify(SimpleCodemod):
         "Makes any calls to requests.{func} with `verify=False` to `verify=True`."
     )
     detector_pattern = """
-        rules:
-          - patterns:
-            - pattern: requests.$F(..., verify=False, ...)
-            - pattern-inside: |
-                import requests
-                ...
+            rules:
+              - pattern-either:
+                - patterns:
+                    - pattern: requests.$F(..., verify=False, ...)
+                    - pattern-inside: |
+                        import requests
+                        ...
+                - patterns:
+                    - pattern: httpx.$F(..., verify=False, ...)
+                    - pattern-inside: |
```

Comment:
I wanted to use metavariable-pattern to just do `$LIBRARY.$F....` but using `import $LIBRARY` in pattern-inside did not work?? weird